python_modules/libraries/dagster-dbt/dagster_dbt/components/dbt_project/scaffolder.py

```diff
@@ -22,20 +22,17 @@ def get_scaffold_params(cls) -> Optional[type[BaseModel]]:
     def scaffold(self, request: ScaffoldRequest, params: DbtScaffoldParams) -> None:
         cwd = os.getcwd()
```

Comment:
what assumptions are there around what `cwd` here? Should I add `project_root` to `ScaffoldRequest` for evaluating the relative path?

Reply:
I don't think there are any assumptions. It it just where the CLI is running from. I doubt this is the behavior we want.